### DISCUSSION TITLE
Cara masukkin Hyperlink di DataTable

### DISCUSSION THREAD

**Discussion by rafiff23** [teamalgoritma/community#605] [Q&A] [answered]
```
df2 <- data.frame(
  URL = c("https://drive.google.com/drive/u/0/folders/1zpSOkHTYVxJMSULwsFPlrXmIqXSHSUfC", "https://drive.google.com/file/d/67890/edit"),
  nama_file = c("file1", "file2")
)
df2$URL <- sprintf('<a href="%s">%s</a>', df2$URL, df2$URL)
datatable(df2, escape =  F)
```

dengan kode saya diatas ini saya sudah berhasil untuk membuat hyperlink di datatable muncul dan bisa diclick, tetapi saya ingin membuat hyperlink tersebut dipersingkat atau diubah menjadi contoh nya "Click Here" dan saat dipencet bisa membuka hyperlink yang sudah didefinisikan.
Saya sudah mencoba nya dengan menambah format style seperti :
```
 formatStyle(
    'URL',
    target = '_blank', 
    style = function(value) {
      htmltools::tags$a(href = value, target="_blank", style = "color: blue; text-decoration: underline;", "Klik disini")
    }
  )
```
tetapi error :
Error in paste(s, collapse = "\n") :
cannot coerce type 'closure' to vector of type 'character'

**Comment by Yusraf002** [comment]
Dear @rafiff23,
Terima kasih atas pertanyaannya pak, tim kami akan segera coba bantu atau carikan referensi ya pak

**Comment by kevbow** [comment, marked as answer]
Dear @rafiff23,

Terdapat 2 referensi terkait penggunaan parameter escape serta contoh yang cukup berguna untuk kasus Anda:
- [RStudio github tentang escape](https://rstudio.github.io/DT/#escaping-table-content)
Pada dokumentasi di atas terdapat contoh penulisan link disertai formatting html yang dapat digunakan untuk menampilkan tulisan yang Anda inginkan, pada kasus ini yaitu "Click Here" sementara di contoh yaitu "RStudio" dan "Hello".
- [Contoh kasus pada datatable](https://stackoverflow.com/questions/32315372/create-a-table-with-clickable-hyperlink)

Demikian yang dapat saya sampaikan, semoga membantu.
